<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVPropBuildingSurfaceConnectionWidget</class>
 <widget class="QWidget" name="SVPropBuildingSurfaceConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <widget class="QGroupBox" name="groupBoxInterlinkedSurfaces">
     <property name="title">
      <string>Interlinked surfaces</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="0" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidgetInterlinkedSurfaces"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonRemoveComponentInstance">
        <property name="text">
         <string>Remove connection</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>280</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxConnectSurfaces">
     <property name="title">
      <string>Connect surfaces</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout2">
      <item>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Maximum distances [m]:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QtExt::ValidatingLineEdit" name="lineEditConnectSurfacesMaxDistance"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Maximum angle [Deg]:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QtExt::ValidatingLineEdit" name="lineEditConnectSurfacesMaxAngle"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButtonConnectSurfaces">
          <property name="text">
           <string>Create connections</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
   <slots>
    <signal>editingFinishedSuccessfully()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
